lib/config/rules/functionality.js

```diff
@@ -0,0 +1,80 @@
+// Functionality Specific:
+// These rules catch common errors in JS programming or
+// otherwise confusing constructs that are prone to producing bugs:
+// Reference https://palantir.github.io/tslint/rules/
+
+module.exports = {
+  // Bans the use of specific functions or global methods.
+  'ban': false,
+  // Enforces braces for `if`/`for`/`do`/`while` statements.
+  'curly': true,
+  // Requires a `for ... in` statement to be filtered with an `if` statement.
+  'forin': true,
+  // Disallows importing the specified modules directly via `import` and `require`.
+  // Instead only sub modules may be imported from that module.
+  'import-blacklist': false,
+  // Only allows labels in sensible locations.
+  'label-position': true,
+  // Disallows use of `arguments.callee`.
+  'no-arg': true,
+  // Disallows bitwise operators.
+  'no-bitwise': false,
```

Comment:
It's not consistent with our eslint config.  @lemonmade how often do we use bitwise ops?